siac/rentercmd.go

```diff
@@ -134,6 +135,16 @@ func renterfilesdownloadcmd(nickname, destination string) {
 	fmt.Printf("Downloaded '%s' to %s.\n", nickname, abs(destination))
 }
 
+// filesize returns a string that displays a filesize in human-readable units.
+func filesizeUnits(size uint64) string {
+	if size == 0 {
+		return "0 B"
+	}
+	sizes := []string{"B", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"}
+	i := int(math.Log10(float64(size)) / 3)
+	return fmt.Sprintf("%.*f %s", i, float64(size)/math.Pow10(3*i), sizes[i])
+}
```

Comment:
Not sure that we ever need more than 2 digits of precision. Having 4 follow the TB seems a bit silly.

It's okay for now though